src/views/tutorial-view/index.tsx

```diff
@@ -258,6 +258,7 @@ function TutorialView({
 					 * deferring for a follow-up PR since this is functional for the time being.
 					 */
 					sidebarNavDataLevels={sidebarNavDataLevels as any}
+					showScrollProgress={true}
```

Comment:
Tutorial view uses the newly introduced `showScrollProgress` prop on `SidebarSidecarLayout`.